.eslintrc.cjs

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+	"plugins": [
+		"evelyn",
+	],
+
+	"extends": [
+		"plugin:evelyn/default",
+		"plugin:evelyn/typescript",
+		"plugin:evelyn/esm",
```

Comment:
Before merging, this config needs to better enforce best practices with importing to prevent easily making super annoying bugs for users